### PR TITLE
docs(i3s): fix formats and english

### PR DESCRIPTION
**Pull request by belom88**

### CODE REVIEW

**Review by dsactionengine — APPROVED** [review]

File: docs/modules/i3s/recipes/attribute-driven-colorization.md [hunk 1]
```diff
@@ -1,118 +1,118 @@
...
+As soon as statistics info is stored in separate resources, it has to be loaded in a separate request. Statistics is just a JSON data and can be loaded with JSONLoader:
+
+```javascript
+import {load} from '@loaders.gl/core';
+import {JSONLoader} from '@loaders.gl/loader-utils';
+
+const attributeStats = await load(`${url}/statistics/f_5/0`, JSONLoader);
+```
+
+## Use I3SLoader with `colorsByAttribute` option
```
Comment (ibgreen) [inline docs/modules/i3s/recipes/attribute-driven-colorization.md:47]: Generally this approach, which recolors values on load, is rather inflexible. Visualization applications often want to let the user change the criteria by which colors are applied using some UI (change the color scale etc). 

I am not saying that we should be in a hurry to remove this feature, but we might want to export a function that generates new colors on already loaded data.

File: docs/modules/i3s/recipes/attribute-driven-colorization.md [hunk 2]
```diff
@@ -1,118 +1,118 @@
...
+        maxValue: statistics.max,
+        minColor: [146, 146, 252, 255], // #9292FC
+        maxColor: [44, 44, 175, 255], // #2C2CAF
+        mode: 'multiply' // or 'replace'
+      }
+    }
+  };
+  return new Tile3DLayer({
+    id: `tile3d-layer-${layer.id}`,
+    data: url,
```
Comment (ibgreen) [inline docs/modules/i3s/recipes/attribute-driven-colorization.md:113]: In the ideal scenario, attribute recolorization should really be a layer prop.

File: docs/modules/i3s/recipes/attribute-driven-colorization.md [hunk 3]
```diff
@@ -1,118 +1,118 @@
...
+
+As soon as statistics info is stored in separate resources, it has to be loaded in a separate request. Statistics is just a JSON data and can be loaded with JSONLoader:
+
+```javascript
+import {load} from '@loaders.gl/core';
+import {JSONLoader} from '@loaders.gl/loader-utils';
+
+const attributeStats = await load(`${url}/statistics/f_5/0`, JSONLoader);
+```
+
```
Comment (ibgreen) [inline docs/modules/i3s/recipes/attribute-driven-colorization.md:46]: More informationb on how colors are loaded in the first place would be good, before we describe recolorization. Right now this is described like a special purpose, black-box feature for deck.gl,

File: docs/modules/i3s/recipes/building-scene-layer.md [hunk 4]
```diff
@@ -1,95 +1,96 @@
...
+  <img src="https://img.shields.io/badge/From-v3.2-blue.svg?style=flat-square" alt="From-v3.2" />
+</p>
+
+A building scene layer can represent 3D model content based on BIM structuring disciplines such as architectural or structural, and categories such as windows or walls. In I3S [specification](https://github.com/Esri/i3s-spec) BSL is a layer type.
+
+BSL doesn't have content resources. It is a composite layer that consists of multiple sublayers. Every sublayer can be: `group`, `3DObject` or `Point`.
+
+`group` sublayer is also just a composite layer. It contains further levels of sublayers and provides a nested structure for BSL. As a result, BSL has a tree-like sublayers structure with `3DObject` and `Point` layers;
+
+`3DObject` and `Point` sublayers are content layers. Those layer types are separated types of layers in I3S spec and can be used independently. Loaders.gl supports `3DObject` layer types and doesn't support `Point`.
```
Comment (ibgreen) [inline docs/modules/i3s/recipes/building-scene-layer.md:13]: These types could perhaps be put in a markdowntable to increase readability?

Here again, more information about these 3DObject layers and what is in them would feel appropriate before we jump into the special case of visualizing them with deck.gl

**Review by ibgreen — COMMENTED** [review]
Really appreciate the added documentation. Below are just some suggestions/thoughts for you to consider for longer term.

Generally, I would like I3S to load into same format as 3D tiles, and these kind of operations should be performed on the general structures in the tiles module, and be documented as general functions on tiles rather than in the I3S module.

Another thing that recently came up in the issues is how to supply a token. That would be good to have a tutorial that explains how to do this for arcgis, google tiles, cesium etc.

**Pull request merged**